vault/testing.go

```diff
@@ -977,6 +977,10 @@ func (c *TestClusterCore) ClusterListener() *cluster.Listener {
 	return c.getClusterListener()
 }
 
+func (c *TestClusterCore) NetworkLayer() cluster.NetworkLayer {
```

Comment:
Hey, I made an identical getter function in a yet-unmerged PR, and wrote this comment out, so I figured I might as well suggest the godoc here 😄

I know that the function itself is super simple, but our overall test cluster machinery is complex, so I think it's nice to clarify intent behind any exported functions for future devs, etc. Feel free to discard this if it's over the top.

```suggestion
// NetworkLayer returns the network layer for the cluster core. This can be used
// in conjunction with the cluster.InmemLayer to disconnect specific nodes from
// the cluster when we need to simulate abrupt node failure or a network
// partition in NewTestCluster tests.
func (c *TestClusterCore) NetworkLayer() cluster.NetworkLayer {
```